consensus/consensus_leader_msg.go

```diff
@@ -46,18 +92,20 @@ func (consensus *Consensus) constructPreparedMessage() ([]byte, *bls.Sign) {
 	// add block content in prepared message for slow validators to catchup
 	consensusMsg.Block = consensus.block
 
-	//// Payload
-	buffer := bytes.NewBuffer([]byte{})
+	// Payload
+	buffer := bytes.Buffer{}
 
 	// 96 bytes aggregated signature
-	aggSig := bls_cosi.AggregateSig(consensus.Decider.ReadAllSignatures(quorum.Prepare))
+	aggSig := bls_cosi.AggregateSig(consensus.Decider.ReadAllSignatures(quorum.Announce))
```

Comment:
why change from Prepare to Announce?

Reply:
So this was a bug?

Reply:
The naming was inconsistent with usage in other parts of consensus and the actual protobuf messages. I wanted to do this in prior time I was around this code but at that time I kept the inconsistent naming